common/src/main/java/io/druid/metadata/MetadataStorageConnector.java

```diff
@@ -44,4 +44,6 @@ public Void insertOrUpdate(
   public void createConfigTable();
 
   public void createTaskTables();
+
+  public void createAuditTable();
```

Comment:
I thought of following benefits of having a separate audit table instead of having an additional column to store the info - 
1) keeping the history of configs separate from the config table seems clean and less confusing to people looking for current configs from db. 
2) makes it easy to have a unified view of all the config changes in the cluster during a time frame.
2) In this format it is easy to integrate with rainer if we choose to go that way later.